docs/plugins/gettingstarted.rst

```diff
@@ -96,15 +96,15 @@ Apart from being discovered by OctoPrint, our plugin does nothing yet. We want t
 
    import octoprint.plugin
 
-   class HelloWorldPlugin(octoprint.plugin.StartupPlugin):
+   class HelloworldPlugin(octoprint.plugin.StartupPlugin):
```

Comment:
'Hello World' is two words, so it should stay as CamelCase for the class name.

Reply:
Fixed 👍